analysis_options.yaml

```diff
@@ -241,6 +242,7 @@ linter:
     - unnecessary_string_interpolations
     - unnecessary_this
     - unnecessary_to_list_in_spreads
+    - unnecessary_underscores
```

Comment:
@pq let me know if this lint is too hot of the press and we shouldn't be enforcing it yet.

Reply:
My hope is it's good to go; feedback is of course totally welcome!

/fyi @kallentu